components/director/pkg/graphql/app_template_validation_test.go

```diff
@@ -561,6 +560,54 @@ func TestApplicationTemplateUpdateInput_Validate_AccessLevel(t *testing.T) {
 	}
 }
 
+func TestApplicationTemplateUpdateInput_Validate_Webhooks(t *testing.T) {
+	webhookInput := fixValidWebhookInput(inputvalidationtest.ValidURL)
+	invalidWebhookInput := fixValidWebhookInput(inputvalidationtest.ValidURL)
+	invalidWebhookInput.OutputTemplate = stringPtr(`{ "gone_status_code": 404, "success_status_code": 200}`)
```

Comment:
Same as above - add a test case for that the URL cannot be `nil` (we were missing this test case in the first place, so nice catch! 💯 )